scripts/get-dep-lib.sh

```diff
@@ -0,0 +1,195 @@
+#!/bin/bash
+#
+
+# set version of 3rd party libraries
+CURRENT_LIBZIP="1.2.0"
+CURRENT_LIBGIT2="v0.26.0"
+CURRENT_HIDAPI="hidapi-0.7.0"
+CURRENT_LIBCURL="curl-7_54_1"
+CURRENT_LIBUSB="v1.0.21"
+CURRENT_OPENSSL="OpenSSL_1_1_0f"
+CURRENT_LIBSSH2="libssh2-1.8.0"
+CURRENT_XSLT="v1.1.29"
+CURRENT_SQLITE="3190200"
+CURRENT_LIBXML2="v2.9.4"
+CURRENT_LIBFTDI="1.3"
+
+
+
+# deal with all the command line arguments
+if [ $# -ne 2 ] ; then
+	echo "wrong number of parameters, format:"
+	echo "get-dep-lib <platform> <install dir>"
+	echo "where platform is one of desktop, ios or android"
+	exit -1
+fi
+
+PLATFORM=$1
+INSTDIR=$2
+if [ ! -d ${INSTDIR} ] ; then
+	echo "creating dir"
+	mkdir -p ${INSTDIR}
+fi
+
+case ${PLATFORM} in
+	desktop)
+		BUILD="libzip libgit2 googlemaps hidapi libcurl libusb openssl libssh2"
```

Comment:
I think that's a bit misleading - the Linux builds and Windows builds don't download these...
This PR doesn't include the intended USE of this script (which is fine), so I'm not sure how you are using this (maybe there's another PR for that already?), but "desktop" seems like a misleading choice.